src/main/resources/application.yml

```diff
@@ -119,6 +119,11 @@ ide-sidecar:
     enabled: true
     # How often to check?
     interval-seconds: 60
+  # Configuration set by the sidecar when instantiating the Kafka Admin client
+  # (We may choose to allow connection-specific overrides in the future)
+  admin-client-configs:
+    "default.api.timeout.ms": 3000
+    "request.timeout.ms": 3000
```

Comment:
@rohitsanj @rhauch What are your thoughts on including a sidecar-specific `client.id`, like `Confluent ide-sidecar ${quarkus.application.version}` (`${quarkus.application.version}` should evaluate to the current release) or similar, in these default configs so that we/users can attribute traffic to the IDE sidecar? This might be especially useful when using the `AdminClient` to interact with remote clusters (CP or CCloud).

Reply:
Yeah, that's a good suggestion. Do we want to just say "sidecar" rather than "VS Code"? Maybe:
```
Confluent for VS Code sidecar ${quarkus.application.version}
```

Reply:
That is indeed a 💯 suggestion, thanks Stefan! I would say we do this in a follow up given the scope and complexity of this PR. I imagine there will be some dynamic string formatting involved to populate the sidecar version in the id string.

Reply:
> Yeah, that's a good suggestion. Do we want to just say "sidecar" rather than "VS Code"? Maybe:
> Confluent for VS Code sidecar ${quarkus.application.version}

I didn't suggest "VS Code" but your proposal works for me 👍

Reply:
> That is indeed a 💯 suggestion, thanks Stefan! I would say we do this in a follow up given the scope and complexity of this PR. I imagine there will be some dynamic string formatting involved to populate the sidecar version in the id string.

I feel this could be a one-liner in the `application.yml` but we can also address it in a follow-up PR if you like.